virttest/qemu_vm.py

```diff
@@ -4001,6 +4007,8 @@ def add_nic(self, **params):
         nic.set_if_none('queues', params.get('queues', '1'))
         if params.get("enable_msix_vectors") == "yes" and int(nic.queues) > 1:
             nic.set_if_none('vectors', 2 * int(nic.queues) + 2)
+        if params.get("romfile") is not None:
```

Comment:
Suggest either `vm_nic_romfile` or `nic_romfile` as the param name for preventing the potential naming collision issue.

Reply:
In fact, I found that romfile had already been introduced, pls refer to https://github.com/avocado-framework/avocado-vt/blob/c40074ba6fe4c089f1418e9b44bae413c12bdcdd/virttest/utils_net.py#L2676, so if we want to change the param name, I am afraid that could lead to unexpected issues in upstream(checked tp-qemu, no romfile was set in tc)

Reply:
@luckyh , what's your opinion? Can we replace romfile with nic_romfile?

Reply:
Actually, I don't understand the background, but some extra params can be appended using `nic_extra_params` or `netdev_extra_params`, so we don't need to introduce new parameters for each option.

Reply:
thanks Yihuang, actually, romfile is not a new param, it is present already, pls refer to my comment above as well as the following:
https://github.com/avocado-framework/avocado-vt/blob/24904ca5335f91a48b31cf6ffce7be287229bc84/virttest/qemu_vm.py#L4135

Background:
For amd sev, we need to disable rom loading by set romfile='', but currently the handling of 'empty string' doesn't work well, so I submitted two commits here to fix it.